build.gradle

```diff
@@ -153,7 +156,17 @@ task renameShadowJar(type: Copy, group: 'release', dependsOn: [shadowJar]) {
 task updateAssets(dependsOn: ['gitCloneAssets', 'gitPullAssets']) {
 }
 
-task allPlatform(type: Zip, group: 'release', dependsOn: [renameShadowJar, updateAssets]) {
+task prepareGameEngineProperties(type: Copy, group: 'release') {
+    inputs.file gameEnginePropertiesFile
+
+    from gameEnginePropertiesFile
+    filter { it.replaceAll(/^(engine_version)\s*=.*$/, "\$1 = $version") }
```

Comment:
Wouldn't it be better to use the ant.propertyfile task to edit the properties file?